src/en/guide/theWebLayer/controllers/commandObjects.gdoc

```diff
@@ -205,3 +205,63 @@ class DemoController {
     }
 }
 {code}
+
+h4. Working with Lists of Command Objects
+
+A common use case for command objects is a Command Object that contains a collection of another:
+
+{code:java}
+class DemoController {
+
+    def createAuthor(AuthorCommand command) {
+        // ...
+
+    }
+
+    class AuthorCommand {
+        String fullName
+        List<BookCommand> books = []
```

Comment:
FYI... `books` doesn't really have to be initialized here.  You could use `List<BookCommand> books` and the binder will initialize it for you.  Do you think that is worth noting?